### PR TITLE
Add option to use PTT as momentary switch

### PR DESCRIPTION
**Pull request by davidAlittle**
Added boolean to allow PTT to function as a momentary switch; same as setting 'stop after' to 1, if that were a feature of PTT.

### CODE REVIEW

**Comment by BarRaider** [comment]
There are a few missing coding standards like using { } correctly
In addition there is already an AutoStopNum that you should probably utilize and set a value of 1

File: SuperMacro/Actions/KeystrokeBase.cs [hunk 1]
```diff
@@ -226,7 +230,9 @@ private void SimulateTextEntry(char character)
 
         private void HandleAutoStop()
         {
-            if (autoStopNum > 0)
+            if (settings.IsMomentary)
+                keyPressed = false;
```
Comment (BarRaider) [inline SuperMacro/Actions/KeystrokeBase.cs:234]: Bad practice

File: SuperMacro/Backend/WriterSettings.cs [hunk 2]
```diff
@@ -22,8 +22,11 @@ public class WriterSettings
 
         public int AutoStopNum { get; private set; }
 
-        public WriterSettings(bool ignoreNewLine, bool enterMode, bool runUntilEnd, bool keydownDelay, bool forcedMacro, int delay, int autoStopNum)
+        public bool IsMomentary { get; private set; }
+
+        public WriterSettings(bool ignoreNewLine, bool enterMode, bool runUntilEnd, bool keydownDelay, bool forcedMacro, bool isMomentary, int delay, int autoStopNum)
```
Comment (BarRaider) [inline SuperMacro/Backend/WriterSettings.cs:27]: Utilize existing autoStopNum

**Comment by davidAlittle** [comment]
Going to close this and open new later.

**Pull request closed without merging**